.github/workflows/reusable-msi-build.yml

```diff
@@ -12,23 +12,23 @@ env:
 
 jobs:
   msi-build:
-    runs-on: ubuntu-24.04
+    runs-on: ${{ inputs.OS }}
     steps:
       - name: Check out the codebase.
         uses: actions/checkout@v5
 
+      - name: Uninstall default WiX
+        run: choco uninstall wixtoolset
+
+      - name: Install WiX 3.14.0
+        run: choco install wixtoolset --version 3.14.0 --allow-downgrade --force
```

Comment:
Is it ok to use an old version? Iirc, you were discussing moving to wix 6 recently